sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -88,6 +97,18 @@
     private static Crashes sInstance = null;
 
     /**
+     * Handler for background thread loop.
+     */
+    @VisibleForTesting
+    final Handler mHandler;
+
+    /**
+     * Handler for main thread loop.
+     */
+    @VisibleForTesting
+    final Handler mMainHandler;
```

Comment:
It was exposed for unit test and now don't need it. Also removing the annotation.